llmproxy/provider/google/vertexai.py

```diff
@@ -3,12 +3,34 @@
 from cohere.client import logger
 from google.cloud import aiplatform
 from vertexai.language_models import TextGenerationModel
+from vertexai.preview.generative_models import GenerativeModel, Part
 
 from llmproxy.provider.base import BaseAdapter
 from llmproxy.utils import logger, timeout, tokenizer
 from llmproxy.utils.enums import BaseEnum
 from llmproxy.utils.exceptions.provider import UnsupportedModel, VertexAIException
 
+from llmproxy.utils.logger import CustomLogger, console_logger, file_logger
+
+# VERTEX IS PER CHARACTER
+vertexai_price_data = {
```

Comment:
The price data is no longer inside of the provider file, it is gathered and passed in through the llmproxy file. So you have to remove this.

Reply:
👍